CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+## [4.0.0] - 2020-11-03
+
+### Added
+
+- Add `ApprovalController` ([#289](https://github.com/MetaMask/controllers/pull/289))
+
+### Changed
+
+- **BREAKING:** Remove `NetworkStatusController` ([#298](https://github.com/MetaMask/controllers/pull/298))
```

Comment:
Nit: There is a `Removed` header for just this purpose :sweat_smile:

Reply:
TIL 😅

Next time!